starport_template/lib/stores/wallets_store.dart

```diff
@@ -45,6 +50,14 @@ class WalletsStore {
 
   set areWalletsLoading(bool val) => Action(() => _areWalletsLoading.value = val)();
 
+  bool get isTransactionHistoryLoading => _isTransactionHistoryLoading.value;
+
+  set isTransactionHistoryLoading(bool val) => Action(() => _isTransactionHistoryLoading.value = val)();
+
+  bool get isTransactionHistoryError => _isTransactionHistoryError.value;
+
+  set isTransactionHistoryError(bool val) => Action(() => _isTransactionHistoryError.value = val)();
+
```

Comment:
the `WalletsStore` is getting really bloated with code. can we extract this code to separate `TransactionsStore` instead?